api/requests.go

```diff
@@ -11,7 +11,7 @@ type GetOrderBookRequest struct {
 	// https://api.valr.com/v1/public/:currencyPair/orderbook
 	// Currency pair
 	// required: true
-	Pair string `json:"currencyPair" url:"currencyPair"`
```

Comment:
Not crucial, but I think it reads better if we can see which annotation is needed.